app/assets/javascripts/discourse/tests/unit/services/screen-track-test.js

```diff
@@ -26,4 +28,18 @@ discourseModule("Unit | Service | screen-track", function () {
       "caches highest read post number for second topic"
     );
   });
+
+  test("appEvent topic:timings-sent is triggered when ..........", async function (assert) {
```

Comment:
Is `…….` supposed to be replaced with something @nattsw ?

Reply:
Oh, I've changed that to "... triggered after posting consolidated timings".

Reply:
@davidtaylorhq yes the exercise is left to the reader. jk -- I had intended it to be replaced based on what @megothss expected, sorry I should have been explicit.